bin/ddg2logo

```diff
@@ -179,14 +185,16 @@ parser = argparse.ArgumentParser(
 description=description, epilog=epilog, formatter_class=argparse.RawTextHelpFormatter)
 optional = parser._action_groups.pop()
 required = parser.add_argument_group('required arguments')
+group = parser.add_mutually_exclusive_group(required=True)
 
 required = init_arguments(['pdb', 'data', 'mutation_list'], required)
 optional = init_arguments(['multimers', 'position_list'], optional)
 
 optional.add_argument("-L","--png-letter-map",dest='pngfile',action='store', default=letters, help="png file to be used for letters")
 
-optional.add_argument("-T","--threshold", dest="threshold", action='store', type=float, default=0.5, help="DDG threshold values - mutations with DDG higher than this are not considered. Default is 0.5")
-optional.add_argument("-x","--maximum-degeneracy", dest="degeneracy", action='store', type=int, default=20, help="when the number of stabilizing mutations is higher than this threshold an X will be plotted instead")
+group.add_argument("-T","--stabthreshold", dest="threshold", action='store', type=float, help="DDG threshold values for stabilizing mutations - mutations with DDG higher than this are not considered. A suggested value could be 0.5")
```

Comment:
can we have words in options separated by dashes, e.g. `--stab-threshold`? same for destab.

also, it would be good to have options with more memorable names than `threshold` and `threshold2`, e.g. `threshold_stab`, `threshold_destab`

Reply:
I added the changes as you suggested